src/osdc/Journaler.cc

```diff
@@ -933,7 +933,7 @@ void Journaler::_assimilate_prefetch()
 
   if (got_any) {
     ldout(cct, 10) << "_assimilate_prefetch read_buf now " << read_pos << "~"
-		   << read_buf.length() << ", read pointers " << read_pos
+		   << read_buf.length() << ", read pointers read_pos/received_pos/requested_pos " << read_pos
```

Comment:
I'd prefer to just convert this to ` read_pos=" << read_pos << " received_pos="...` to ease readability.